zjit/src/hir.rs

```diff
@@ -1309,6 +1310,25 @@ impl Function {
         }
     }
 
+    fn try_rewrite_subscript(&mut self, block: BlockId, orig_insn_id: InsnId, self_val: InsnId, idx_val: InsnId) {
```

Comment:
```suggestion
    fn try_rewrite_aref(&mut self, block: BlockId, orig_insn_id: InsnId, self_val: InsnId, idx_val: InsnId) {
```

`#[]` is called `aref` very consistently in the CRuby codebase, so calling it differently could confuse contributors, at least including myself.